compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeDialog.desktop.kt

```diff
@@ -112,20 +138,15 @@ class ComposeDialog : JDialog {
     constructor(
         graphicsConfiguration: GraphicsConfiguration? = null,
     ) : this(
-        owner = null,
-        modalityType = ModalityType.MODELESS,
+        owner = null as Frame?,
         graphicsConfiguration = graphicsConfiguration,
         skiaLayerAnalytics = SkiaLayerAnalytics.Empty,
         savedState = null
     )
 
-    // don't replace super() by super(null, ModalityType.MODELESS), because
-    // this constructor creates an icon in the taskbar.
-    // Dialog's shouldn't be appeared in the taskbar.
-    constructor() : super() {
```

Comment:
Removing this needs to be checked by @igordmn

Reply:
The important part is not to call `super()`, but to call a `JDialog` constructor that uses `SwingUtilities.getSharedOwnerFrame()` as the owner if the given owner is `null`. All `JDialog` constructors that take `JFrame?` do this.

Reply:
Yes, seems correct calling the Frame accepting constructor as well, verified it by checking manually and looking at the sources.